doc/input-output-reference/src/overview/group-air-distribution-equipment.tex

```diff
@@ -1758,6 +1763,10 @@ \subsubsection{Inputs}\label{inputs-12-000}
 
 This input field allows user to specify the mixer connection type. Valid choices are \textbf{InletSide} or \textbf{SupplySide}. This is a required input field. If the mixer connection type selected is \textbf{InletSide}, then the mixer is connected on the inlet side of the ZoneHVAC equipment, or else if the mixer connection type selected is \textbf{SupplySide}, then the mixer is connected at the outlet side of the ZoneHVAC equipment.\\ 
 
+\paragraph{Field: Design Specification Outdoor Air Object Name}\label{field-DSOA-object-name}
+
+This field allows modifying the behavior of this air terminal so that it is modulated to supply the required outdoor air to the zone.  This field is optional. When the name of an DesignSpecification:OutdoorAir object is entered, the model is changed to adjust the flow rate to provide the volume of outdoor air described by that object.  This feature allows modeling demand controlled ventilation on a zone-by-zone basis using the Outdoor Air Flow per Person rate (specified in the DesignSpecification:OutdoorAir object) and the number of occupants (specified in the People object schedules).  If the outdoor air fraction of the supply air is 1.0, as for a dedicated outdoor air system, the air flow rate will match the outdoor air requirement.  When the outdoor air fraction is less than 1.0, as for a recirculating air system, the terminal air flow will be modulated upward to account for the increased total air flow needed to provide the required flow rate of outdoor air.   The total air flow rate will not exceed the Maximum Air Flow Rate specified above. The volume flow rate is converted to mass flow rate using the current density of air (function of drybulb and humidity ratio) at the supply air node.
+
```

Comment:
Same comment about density here.  Also, this needs to say that the DSAO object from Sizing:Zone will be used if this field is blank.  And if neither is present, then the OA flow will be zero.